### ISSUE TITLE
"https://teams.microsoft.com/_" not loading

### ISSUE THREAD

**Issue by davidfarah2003** [TobiasPankner/Teams-Auto-Joiner#78]
**Describe the bug**
I set up everything in Pycharm and installed the requirements, however, after logging in to the selenium browser it gets stuck on "https://teams.microsoft.com/_" loading indefinitely until timeout. I tried multiple things like using undetected_chromedriver and firefox geckodriver with no results. I even tried to remove the options added, type account manually, and use a VPN, nothing worked. The website works fine in the normal browser.

**To Reproduce**
1. open pycharm
2. Create a new project
3. copy the script and files to the directory
4. Set up the interpreter etc...
5. run the script

**Expected behavior**
It's expected to log into my account and continue to the teams main page

**Screenshots**
![image](https://user-images.githubusercontent.com/37240868/109136373-b7412e00-7760-11eb-9134-a3ba92b34ea0.png)

**Configuration file**
{
  "email": "MYEMAIL",
  "password": "MYPASS",

  "run_at_time": "",

  "meeting_mode": 2,
  "organisation_num" : 1,
  "random_delay": true,
  "check_interval": 10,

  "join_message": "",

  "auto_leave_after_min": -1,
  "leave_if_last": true,
  "leave_threshold_number": "5",
  "leave_threshold_percentage": "",
  "pause_search": false,

  "headless" : false,
  "mute_audio": false,
  "chrome_type": "chromium",

  "blacklist": [
    {
      "team_name": "Example Team 1",
      "channel_names": [
        "First Channel to blacklist in this team",
        "Second Channel to blacklist"
      ]
    },
     {
      "team_name": "Blacklist whole team",
      "channel_names": [
      ]
    }
  ],
  "blacklist_meeting_re": ""
}

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome and edge
 - Version: 88.0.4324.190

**Additional context**

**Comment by TobiasPankner** [comment]
Can you try with a different website to determine if it's specific to Teams?

**Comment by davidfarah2003** [comment]
I tried it with google chrome it gave me a prompt that the browser is not secure.
here is the screenshot:
![image](https://user-images.githubusercontent.com/37240868/109138887-88788700-7763-11eb-8597-d25206a3292f.png)

I tried it now with normal python in command prompt and it loaded teams after some time.

**Comment by davidfarah2003** [comment]
I got it working now thanks for the reply. However, it's not joining the meeting in Calendar mode (mode 3).  And is it possible to check the calendar once then memorize the upcoming meetings? because I want to use it to record the sessions but it's constantly exiting them.

**Comment by TobiasPankner** [comment]
No, that is not currently possible. Is there a active meeting in the calendar to join?

**Comment by TobiasPankner** [comment]
You can change pause_search to true to avoid the exiting

**Comment by davidfarah2003** [comment]
Yes, there is an active meeting and it's working with the mode "channel meetings". Thanks for the tip on exiting. Is there a way to prevent the people tab from opening and closing or keep it open?

**Comment by TobiasPankner** [comment]
It needs to be open to check for members, right now there is no way to keep it open. I'll take a look at this later.  
And just to clarify, does it work with mode 1 or 2?

**Comment by davidfarah2003** [comment]
Yes, the detection does work in modes 1 and 2 through the Channel meetings. Btw I am currently working on the member counting without opening it.

**Comment by TobiasPankner** [comment]
Great, just create a pull request when it works.

**Comment by davidfarah2003** [comment]
I messed around a little bit yesterday, the contents of the menu are not accessible when it's closed. However, it should still be pretty easy to do by using the users that show up in rectangles under the meeting in the default view. Haven't tried yet because I don't have an unimportant meeting today where I can test.

**Comment by Fafera42** [comment]
Hello! I know I'm late to this thread but I was wondering on how you fixed the errors you get in that CMD screenshot you posted, I'm experiencing the same exact error codes on the CMD and is clueless on how to really fix it, would really appreciate if someone could help me out here. :)

**Comment by Fafera42** [comment]
My chrome browser + CMD crashes after those errors
